polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -97,67 +58,47 @@ const formatTokenGroup = (tokenGroup: TokenGroup, format: Format) => {
       .join('\n');
   }
 
-  return tokenGroup;
+  return tokenValues;
 };
 
 const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   const formatParam = (req.query.format || 'json') as Format;
-  const schemeParam = (req.query.scheme || 'light') as Scheme;
+  const schemeParam = (req.query.scheme || 'light') as ColorScheme;
```

Comment:
```suggestion
  const schemeParam = isScheme(req.query.scheme) ? req.query.scheme : 'light';
```

Same as above. This PR removes the `isScheme` type guard which safely checks and casts the `query.scheme` param. Thoughts?